indexer_app/utils.py

```diff
@@ -744,7 +744,6 @@ async def handle_payout_challenge(
         payoutChallenge = {
             "created_at": created_at,
             "message": data["reason"],
-            "tx_hash": receiptId,
```

Comment:
this should not be removed; rather, a `tx_hash` field should be added to `PayoutChallenge`